website/src/components/canvas/CanvasList.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file HTML generation for the canvas list
+ */
+import React from 'react';
+import { Box, Flex } from 'reflexbox';
+import { Card, CardHeader, CardMedia } from 'material-ui/Card';
+import { Link } from 'react-router';
+
+const styles = {
+  models: {
+    display: 'inline-block',
+    margin: 15,
+  },
+};
+
+const generateCanvasCode = (
+    <img src="https://res.cloudinary.com/craftml/image/upload/w_250,h_250,c_fill/v1440024165/4yUaf.png" className='img-responsive' />
+ );
+
+/**
+ * Generates HTML for the user canvas list.
+ * @returns {canvasList}  The array holding the canvas list HTML.
+ */
+function CanvasList() {
+  //TODO
+  //Add firebase call to get canvas IDs and img src
+  let canvasList = [];
+  let numCanvases = 10;
+  let numCols = 3;
+  if (numCanvases > 5){
```

Comment:
Why are there two checks for if >5 and >10? You could just remove the >5 if statement since nothing happens if numCanvases > 5